### PR TITLE
[strings] Fixed iCloud

### PR DESCRIPTION
**Pull request by matheusgomesms**
Fixed PT and PT-BR strings after #7641.

### CODE REVIEW

File: iphone/Maps/LocalizedStrings/pt-BR.lproj/Localizable.strings [hunk 1]
```diff
@@ -1302,7 +1302,7 @@
 "enable_icloud_synchronization_title" = "Ativar a sincronização do iCloud";
 
 /* Message for the "Enable iCloud Syncronization" alert. */
-"enable_icloud_synchronization_message" = "A sincronização do iCloud é um recurso experimental em desenvolvimento. Certifique-se de que você tenha feito um backup de todos os seus favoritos e faixas.";
+"enable_icloud_synchronization_message" = "A sincronização do iCloud é um recurso experimental em desenvolvimento. Certifique-se de que você tenha feito um backup de todos os seus favoritos e trilhas.";
```
Comment (biodranik) [inline iphone/Maps/LocalizedStrings/pt-BR.lproj/Localizable.strings:1305]: Thanks! Please also fix translations in https://github.com/organicmaps/organicmaps/blob/master/data/strings/strings.txt

Comment (matheusgomesms) [reply iphone/Maps/LocalizedStrings/pt-BR.lproj/Localizable.strings:1305]: Yep, I made a mess, so I had to do it now manually. Can you please rebase and combine multiple commits into one, please?

**Comment by matheusgomesms** [comment]
Sorry, I totally forgot about strings.txt and strings regen, so in the end I did it manually (using web version). No problem, since it was just a couple of strings.

Can you please rebase and combine multiple commits into one, please?

File: iphone/Maps/LocalizedStrings/pt-BR.lproj/Localizable.strings (hunk 1 above)
Comment (biodranik) [reply iphone/Maps/LocalizedStrings/pt-BR.lproj/Localizable.strings:1305]: Done!

**Review by biodranik — APPROVED** [review]

**Pull request merged**